testing/tests/DevExpress.ui.widgets/overlay.tests.js

```diff
@@ -2945,6 +2946,20 @@ QUnit.test("elements under top overlay with shader have not to get focus by tab"
     assert.equal(this.tabEvent.isDefaultPrevented(), false, "default action is not prevented");
 });
 
+QUnit.test("tabbable selectors should check only bounds", function(assert) {
+    var tabbableSpy = sinon.spy(selectors, "tabbable");
+    var overlay = new Overlay($("<div>").appendTo("#qunit-fixture"), {
+        visible: true,
+        shading: true,
+        contentTemplate: $("#focusableTemplate")
+    });
+    var elementsCount = $(overlay.content()).parent().find("*").length;
+
+    overlay._findTabbableBounds();
```

Comment:
Don't use the private method. Use keyboard mock, press tab and check the count of tabbable method calls